Berlin/BerOrOct59.xml

```diff
@@ -26,43 +30,250 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+
+                 <physDesc>
+                    <objectDesc form="Codex">
+                       <supportDesc>
+                        <support>
+                           <material key="paper"/>
+                            </support>
+
+                    <foliation>
+                        <list>
+                         <item resp="p1">
+                          <locus from="1r" to="4r"/>
+                          Foliation in pencil in the upper right corner.
+                           </item>
+                        </list>
+                     </foliation>
+                     </supportDesc>
+                   </objectDesc>
+
+                 <handDesc>
+                    <handNote xml:id="h2" script="Latin" corresp="p1_i1">
+                      <locus from="1r" to="4v"/>
+                    <date when="1692" evidence="internal"/>
+                      <persName role="scribe" ref="PRS6378Ludolf"></persName>
+                      <desc> Handwritten description of the content of the manuscript by <persName role="scribe" ref="PRS6378Ludolf"></persName>) himself.</desc>
+                    </handNote>
+                    </handDesc>
+                    </physDesc>
            </msPart>
-           <msPart xml:id="p2">
+
+         <msPart xml:id="p2">
            <msIdentifier> <idno>SBP, Ms. orient. oct. 59, p2</idno>
-         </msIdentifier>
+           </msIdentifier>
                <msContents>
                    <msItem xml:id="p2_i1">
-                      <msItem xml:id="ms_i1">
                         <locus from="1r" to="11v"></locus>
-                        <title> Magic prayer against caugh </title>
+                        <title> Magic prayer against caugh. </title>
                         <textLang mainLang="gez"></textLang>
+                        <incipit xml:lang="gez"> <supplied reason="lost">በሰመ፡</supplied> <hi rend="rubric"> አብ፡ ወወልድ፡ወመንፋስ፡ ቅ</hi>
+                        አሐዱ፡ አምላክ። ጸሎት፡ ወ<hi rend="rubric">ስእለት፡ ወአስተብቊዖት፡ <gap reason="lost"/> </hi> ት፡ ዘእግዝእትነ፡ ማርዛም፡
+                        </incipit>
                       </msItem>
-                 <msItem xml:id="ms_i2">
+                 <msItem xml:id="p2_i2">
                    <locus from="15r" to="55r"></locus>
                    <title ref="LIT1201Bartos"/>
                    <textLang mainLang="gez"></textLang>
                  </msItem>
-                 <msItem xml:id="ms_i3">
-                   <locus from="15r" to="55r"></locus>
-                   <title />
+                 <msItem xml:id="p2_i3">
+                   <locus from="57r" to="65v"></locus>
+                   <title> Marian prayer. </title>
                    <textLang mainLang="gez"></textLang>
+                   <incipit xml:lang="gez"><hi rend="rubric"> በስመ፡ አብ፡ ወወልድ፡ወመን</hi>ፈስ፡ ቅዱስ፡ አሐዱ፡ አምላ<hi rend="rubric">
+                   ክ። ወበውእቱ፡ መዋዕል፡ አን</hi> ሥአት፡ ማርዛም፡ ወሖረት፡ <hi rend="rubric">ሓቅለ፡ ወቦአት፡ ቤቶ፡ ለዘ</hi>ካርያስ፡ ደወለ፡ ዐይነ፡ ካርም፡
```

Comment:
`አን</hi> ሥአት፡` will be printed with a space after n - if you don't want this, delete the space here
also throughout